ngi_reports/ngi_reports.py

```diff
@@ -103,7 +105,7 @@ def make_reports(report_type, working_dir=os.getcwd(), config_file=None, **kwarg
     # Load the Jinja2 template
     try:
         env = jinja2.Environment(loader=jinja2.FileSystemLoader(reports_dir))
-        template = env.get_template("{}.md".format(report_type))
+        template = env.get_template("project_summary.md")
```

Comment:
Here too? Only use `project_summary.md` if report_type is 'project_summary'?

Reply:
Done